src/documents/tutorial-userInterfaceOptions/UserInterfaceOptions.md

```diff
@@ -66,7 +73,10 @@ Add the following markup as the first thing within your `<body>` tag:
 </div>
 ```
 
-The main `<div>` in this snippet contains two things:
+The main `<div>` in this snippet contains 
+
+
+things:
 
 1. a `<div>` where an iframe will be inserted, containing the Fat Panel UI Options controls, and
```

Comment:
"Fat Panel UI Options controls" should be renamed to just "UI Options adjusters"